setup.py

```diff
@@ -34,6 +34,7 @@ def read(*names, **kwargs):
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
```

Comment:
needs Python 2, otherwise programs might assume 'Programming Language :: Python :: 3' without 'Programming Language :: Python :: 2' implies 'Programming Language :: Python :: 3 only'